riscv.c

```diff
@@ -863,25 +1417,37 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
 #undef EXEC
 #undef TARGET
 #else   // ENABLE_COMPUTED_GOTO = 0
+    inst = 0;
     while (rv->csr_cycle < cycles_target && !rv->halt) {
         // fetch the next instruction
         inst = rv->io.mem_ifetch(rv, rv->PC);
 
+        // Illegal instruction if inst[15:0] == 0
+        // assert(inst & 0xFFFF && "inst[15:0] must not be all 0.\n");
```

Comment:
Would you rework the illegal instruction handling?

Reply:
If the illegal instruction occur, that will call rv_except_illegal_inst function:
1. record PC by csr_mepc
2. record instruction by csr_mtval
3. record except code by mscause
4. jump to base(csr_mtvec & ~0x3)

Reply:
You can add/improve comments around the illegal instruction handling in source files.